packages/backend/src/payment-method/ilp/connector/core/middleware/balance.ts

```diff
@@ -110,6 +119,7 @@ export function createBalanceMiddleware(): ILPMiddleware {
         } else {
           await trx.void()
         }
+        stopTimer()
```

Comment:
I think need to include `stopTimer()` also on L114 and L108

Reply:
Reworked it to not have all those `stopTimer()`s.